src/Erl/Data/Map.purs

```diff
@@ -1,105 +1,33 @@
-module Erl.Data.Map
+module Erl.Data.Map (
+  module Erl.Data.Map.Internal
```

Comment:
I get this error when
`> erl -pa ebin -noshell -eval '(test_main@ps:main())()' -eval 'init:stop()'
`
```
/Users/cmdv/Dev/PSErl/purescript-erl-maps/ebin/erl_data_map_internal@foreign.beam: Module name 'erl_data_map@foreign' does not match file name 'erl_data_map_internal@foreign'
make: *** [Makefile:13: erl] Error 1
``` 

then when looking in erl_data_map@ps.erl

```
% Generated by purs version 0.0.1
-module(erl_data_map@ps).
-export([]).
-compile(nowarn_shadow_vars).
-compile(nowarn_unused_vars).
-compile(no_auto_import).
```

it seems to have worked in Pinto so not sure why it's doing this here as of yet.